### ISSUE TITLE
[Feature Request] Storage/Quota Visualisation

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#10859]
I didn't see an issue for this, so, figured I'd open one. It's a common complaint I've heard of "what's taking up space" and "what can I delete." Currently we point people to two off-galaxy resources (the hub, the gtn) for tutorials but what users need is not a tutorial. They need their quota bar to link to a list of their histories, and the associated sizes. Maybe with a delete button.

I've privately heard numerous proposals for tree views and archival and other things, but, the core issue is that we shouldn't be pushing people outside of galaxy to a tutorial, we should be showing them their data/usage and giving them a delete button. They really don't need a tutorial on "move to recycle bin".

**Comment by bernt-matthias** [comment]
Sounds good. Maybe the list of histories that can be accessed from the user menu, but sorted by size.

**Comment by davelopez** [comment]
xref https://github.com/galaxyproject/galaxy/pull/13113

**Comment by hexylena** [comment]
Fixed in #13113

**Comment by mvdbeek** [comment]
This is just for already deleted datasets that can be purged for now. It is modular, but I think we should only close this when we can also select non-deleted datasets (but that's a little tricky, since you don't want to accidentally destroy collections or published data)

**Comment by hexylena** [comment]
Yeah, ok, that's fair based on my original text. I was too excited there for a second :)

**Comment by hexylena** [comment]
Honestly this issue would be solved by having dataset sizes in the /datasets/list view that I always forget exists. I know that doesn't address collections but, it could? maybe?
